MIDIchlorians/MIDIchlorians/Session.swift

```diff
@@ -53,6 +53,36 @@ class Session: Object {
         }
     }
 
+    convenience init?(json: Data) {
+        self.init()
+        guard let dictionary = (try? JSONSerialization.jsonObject(with: json, options: [])) as? [String: Any?] else {
+            return nil
+        }
+        guard let BPM = dictionary["BPM"] as? Int,
```

Comment:
Magic constants.. but I guess we can ignore these for now and look for them later when we clean up our code after steps :)